summoner-cli/src/Summoner/CLI.hs

```diff
@@ -93,7 +93,7 @@ Available commands:
 runShow :: ShowOpts -> IO ()
 runShow = \case
     -- show list of all available GHC versions
-    GhcList -> showBulletList @GhcVer showGhcVer (reverse universe)
+    GhcList -> showBulletList @GhcVer showGhcOutput (reverse universe)
```

Comment:
Regarding proper alignment you can do the following thing:

1. Change the resulting type of `showGhcOutput` function to some tuple.
2. Have a function:
    ```haskell
    alignTable :: [(Text, Text, Text)] -> [Text]
    ```
    This function will format the list properly by finding the maximum length of each component and adding required number of spaces.

Then you can easily combine it with the existing `showBulletList` :slightly_smiling_face: